checkstyle/spotbugs-include.xml

```diff
@@ -48,7 +46,7 @@
     <!-- Experimental and not fully vetted bug patterns -->
     <Match>
         <Bug category="EXPERIMENTAL" />
-        <Rank value="9"/>
+        <Confidence value="1"/>
```

Comment:
Is confidence equivalent of rank? what does value 1 means?

Reply:
No confidence used to be priority in prior versions. It indicates a confidence in the possibility that spotbugs is correct for that bug type. It indicates the confidence that the found bug is actually a bug. FindBugs employs heuristics which may go wrong. It varies from 1 (highest confidence) to 5 (lowest confidence, to be disregarded). 

Rank indicates how critical a bug is or in spotbugs terms how "scary" it is. It indicate the impact of that bug on the system - how severe it would be.  The values range this way:
Scariest (1-4),
Scary (5-9),
Troubling (10-14)
Of Concern (rank 15-20)
One analog is; So, a certain NullPointerException might get a 1, and a potential performance improvement might just get a 12.

The values match those in Pravega spot bugs. The expected time for me to do this was evidently short and Tom had converted the Pravega build, so I trusted his judgement, since I didn't have the leeway to investigate each of the potential spotbugs type.